### DISCUSSION TITLE
port 443 busy at reboot

### DISCUSSION THREAD

**Discussion by gualtierogrey** [AzuraCast/AzuraCast#7244] [General]
hi, i have azuracast installed on a home server in docker. low specs but it does the job for the time being. got a few other apps running such as home assistant and zone minder which is still working in the background for recording but won't display ui anymore as i've unistalled apache 2. problem is, every time i reboot azuracast won't restart, as the port 443 seems busy

Error starting userland proxy: listen tcp 0.0.0.0:443: bind: address already in use

when i run 

netstat -ltnp | grep ':443'

it shows 

tcp        0      0 0.0.0.0:443             0.0.0.0:*               LISTEN      1027/nginx: master
tcp6       0      0 :::443                  :::*                    LISTEN      1027/nginx: master

killing that process and restarting azuracast with 

docker-compose up -d

frees the the port and the radio is back online. 

Any idea why this happens? looked around but still can't find a solution. many thanks.

**Comment by gAlleb** [comment]
Hi @satellouk! This means that `nginx` is running on your system. You can disable it from starting on reboot with `systemctl disable nginx`. To stop nginx instead of killing it - `systemctl stop nginx`.

To see the status: `systemctl status nginx`.  By default nginx takes up ports 80 and 443. Maybe you've installed it and then forgot about it)

**Reply by gualtierogrey** [reply]
thanks brother.. done so many things on this system since installing ubuntu server back in 2018 ..
